src/steadystateproblem.cpp

```diff
@@ -303,87 +316,54 @@ void SteadystateProblem::getSteadystateSimulation(ReturnData *rdata, Solver *sol
         if (it_newton >= solver->getMaxSteps())
             throw NewtonFailure(AMICI_TOO_MUCH_WORK,"getSteadystateSimulation");
     }
+    if (it<1) {
+        if (newtonSimSolver->getSensitivityOrder()>SensitivityOrder::none)
```

Comment:
```
if (newtonSimSolver->getSensitivityOrder() > SensitivityOrder::none)  {
  if(it < 1) ...
else ...
} 
```

Reply:
nope `newtonSimSolver->getSensitivityOrder()` and `dolver->getSensitivityOrder()` may differ

Reply:
but probably can combine two with && and use elseif

Reply:
Ah, true. Missed that.